File: Assignments/Assignment_01/Task_03.java
    public static int[] shiftRight(int[] source, int k){
        int limit = k;
        for (int i=0; i<limit; i++,k++){
            if (k<source.length){
                source[(k)] = source[i];
                source[i] = 0;
            }
            else{
                source[i] = 0;
            }
        }
        return source;
    

File: Assignments/Assignment_01/Spring-23/Task_03.py
# Test 03: Shift Right k cell
def shift_right(source, k):
 
  # Hint, You can write a function for right shift once and then use it
  j = len(source)-1
  while j>=0:
      if j>=k:
        source[j] = source[j-k]
      else:
        source[j] = 0
      j -= 1

  return source


